docs/api-docs/apps/tutorials/sample-app-next/05-step-3-integrate-api.md

```diff
@@ -0,0 +1,920 @@
+
+```js
+import { NextApiRequest, NextApiResponse } from 'next';
+import { encodePayload, getBCVerify, setSession } from '../../lib/auth';
+```
+
+3. Update the logic to authenticate the app on load.
+
+```js
+export default async function load(req: NextApiRequest, res: NextApiResponse) {
+    try {
+        // Verify when app loaded (launch)
+        const session = await getBCVerify(req.query);
+        const storeHash = session?.context?.split('/')[1] || '';
+        const encodedContext = encodePayload(storeHash); // Signed JWT to validate/ prevent tampering
+
+        await setSession(session);
+        res.redirect(302, `/?context=${encodedContext}`);
+    } catch (error) {
+        const { message, response } = error;
+        res.status(response?.status || 500).json(message);
+    }
+}
+```
+
+[View code in GitHub](https://github.com/bigcommerce/sample-app-nodejs/blob/step-3-add-database/pages/api/load.ts)
+
+**uninstall** 
+
+1. Open the `uninstall.ts` file nested inside the `pages/api` folder.
+
+2. Import `getBCVerify` and `removeSession` from `/lib/auth`. Your imports should now look like this:
+
+```js
+import { NextApiRequest, NextApiResponse } from 'next';
+import { getBCVerify, removeSession } from '../../lib/auth';
+```
+
+3. Update the logic to delete the session on uninstall.
+
+```js
+export default async function uninstall(req: NextApiRequest, res: NextApiResponse) {
+    try {
+        const session = await getBCVerify(req.query);
+
+        await removeSession(res, session);
+        res.status(200).end();
+    } catch (error) {
+        const { message, response } = error;
+        res.status(response?.status || 500).json(message);
+    }
+}
+```
+
+[View code in GitHub](https://github.com/bigcommerce/sample-app-nodejs/blob/step-3-add-database/pages/api/uninstall.tss)
+
+## Add the Products endpoint
+
+You will use this endpoint to retrieve your products summary.
+
+1. In the `pages/api` folder, create a new folder called `products`.
+
+2. In the `products` folder, create an `index.ts` file. This will create a `/products` route.
+
+3. At the top of the file, import the following packages:
+
+```js
+import { NextApiRequest, NextApiResponse } from 'next';
+import { bigcommerceClient, getSession } from '../../../lib/auth';
+```
+
+4. Add the async `products` function, which will await the data returned from `bigcommerce.get`.
+
+```js
+export default async function products(req: NextApiRequest, res: NextApiResponse) {
+    try {
+        const { accessToken, storeHash } = await getSession(req);
+        const bigcommerce = bigcommerceClient(accessToken, storeHash);
+
+        const { data } = await bigcommerce.get('/catalog/summary');
+        res.status(200).json(data);
+    } catch (error) {
+        const { message, response } = error;
+        res.status(response?.status || 500).end(message || 'Authentication failed, please re-install');
```

Comment:
sorry, this was another place we recently updated... this should be just 
`res.status(response?.status || 500).json({ message });`